src/Middleware.ts

```diff
@@ -616,9 +626,12 @@ export function json<E>(
  * @category constructors
  * @since 0.7.0
  */
-export function redirect<E = never>(uri: string | { href: string }): Middleware<StatusOpen, HeadersOpen, E, void> {
+export function redirect<E = never>(
+  uri: string | { href: string },
+  code: RedirectionStatus = Status.Found
```

Comment:
`Location` can also be used on a `201 Created` response, but I'm not sure if that should be included here or not.